prefetch.bs

```diff
@@ -129,48 +130,73 @@ A <dfn>redirect chain</dfn> is a [=list=] of [=exchange records=].
 
 <hr>
 
-Each {{Document}} has a <dfn export>prefetch buffer</dfn>, which is a [=list=] of [=prefetch records=].
+Each {{Document}} has <dfn export for="Document">prefetch records</dfn>, which is a [=list=] of [=prefetch records=].
 
-A <dfn>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
+A <dfn export>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
 * <dfn export for="prefetch record">URL</dfn>, a [=URL=]
 * <dfn export for="prefetch record">referrer policy</dfn>, a [=referrer policy=]
-* <dfn export for="prefetch record">sandbox flags present</dfn>, a boolean
-* <dfn export for="prefetch record">redirect chain</dfn>, a [=redirect chain=]
-* <dfn export for="prefetch record">expiry time</dfn>, a {{DOMHighResTimeStamp}}
+* <dfn export for="prefetch record">anonymization policy</dfn>, a [=prefetch IP anonymization policy=]
+* <dfn export for="prefetch record">initiator</dfn>, a {{DOMString}}
+
+  <div class="note">This is intended for use by a specification or [=implementation-defined=] feature to identify which prefetches it created. It might also associate other data with this struct.</div>
+* <dfn export for="prefetch record">state</dfn>, which is `"ongoing"` (the default), `"completed"`, or `"canceled"`
```

Comment:
Nit here and elsewhere, we use backticks inside the quote marks for spec string literals.

Reply:
Done.